extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -189,8 +176,12 @@
   right: 0;
 }
 
+.amp-lbg-icon {
```

Comment:
Using `mask-image` and `background-color` allows us to change the background color without changing the icon.